### PR TITLE
fix: add clarity to parameters per user feedback

### PR DESCRIPTION
**Pull request by andrewferlitsch**
Thank you for opening a Pull Request! Before submitting your PR, there are a few things you can do to make sure it goes smoothly:
- [x] Make sure to open an issue as a [bug/issue](https://github.com/googleapis/python-aiplatform/issues/new/choose) before writing your code!  That way we can discuss the change, evaluate designs, and agree on the general idea
- [x] Ensure the tests and linter pass
- [x] Code coverage does not decrease (if any source code was changed)
- [ ] Appropriate docs were updated (if necessary)

Fixes #<issue_number_goes_here> 🦕

### CODE REVIEW

File: samples/model-builder/predict_tabular_classification_sample.py [hunk 1]
```diff
@@ -22,9 +22,17 @@
 def predict_tabular_classification_sample(
     project: str, location: str, endpoint: str, instances: List[Dict],
 ):
+    '''
+    Args
+        project: Your project ID or project number.
+        location: Region where Endpoint is located. For example, 'us-central1'.
+        endpoint: A fully qualified endpoint name or endpoint ID. Example: "projects/123/locations/us-central1/endpoints/456" or
```
Comment (morgandu) [inline samples/model-builder/predict_tabular_classification_sample.py:29]: Docstring also need to reflect the change from `endpoint` to `endpoint_name`

File: samples/model-builder/predict_tabular_classification_sample.py [hunk 2]
```diff
@@ -22,9 +22,17 @@
 def predict_tabular_classification_sample(
     project: str, location: str, endpoint: str, instances: List[Dict],
```
Comment (morgandu) [inline samples/model-builder/predict_tabular_classification_sample.py:23]: based on the change of this PR, `endpoint` should also be changed to `endpoint_name`

**Review by morgandu — CHANGES_REQUESTED** [review]

File: samples/model-builder/predict_tabular_classification_sample.py (hunk 1 above)
Comment (andrewferlitsch) [reply samples/model-builder/predict_tabular_classification_sample.py:32]: fixed

File: samples/model-builder/predict_tabular_classification_sample.py (hunk 2 above)
Comment (andrewferlitsch) [reply samples/model-builder/predict_tabular_classification_sample.py]: fixed

File: samples/model-builder/predict_tabular_classification_sample.py (hunk 1 above)
Comment (andrewferlitsch) [reply samples/model-builder/predict_tabular_classification_sample.py]: fixed

**Review by morgandu — APPROVED** [review]

**Pull request merged**